WhaleSpotting/Models/Request/UserRequest.cs

```diff
@@ -6,4 +6,5 @@ public class UserRequest
     public string Password { get; set; }
     public string ProfileImageUrl { get; set; }
     public string UserBio { get; set; }
+    public UserType UserType { get; set; }
```

Comment:
Did you test that this was working? I think there will be an issue converting from the front to backend. It is probably easier to make this a string and then have some logic to handle `if(UserType == "Member")`